src/libs/actions/IOU.ts

```diff
@@ -1728,7 +1735,7 @@ function createSplitsAndOnyxData(
         // For Control policy expense chats, if the report is already approved, create a new expense report
         let oneOnOneIOUReport: OneOnOneIOUReport = oneOnOneChatReport.iouReportID ? allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${oneOnOneChatReport.iouReportID}`] : null;
         const shouldCreateNewOneOnOneIOUReport =
-            !oneOnOneIOUReport || (isOwnPolicyExpenseChat && ReportUtils.isControlPolicyExpenseReport(oneOnOneIOUReport) && ReportUtils.isReportApproved(oneOnOneIOUReport));
+            !oneOnOneIOUReport || (isOwnPolicyExpenseChat && ReportUtils.isPaidGroupPolicy(oneOnOneIOUReport) && ReportUtils.isReportApproved(oneOnOneIOUReport));
```

Comment:
We previously only supported Control type of workspaces for splits. This logic checks whether or not we can add a new expense (designated IOU here) which is created from a split to an existing report or create a new one. If the report is approved (approvals only exist in paid policies, hence the `isPaidGroupPolicy` check), we add the new expense to a new report. Makes sense?

Reply:
>Just wanna double check whether we should use ReportUtils.isPaidGroupPolicyExpenseReport instead of ReportUtils.isPaidGroupPolicy here.

It doesn't really matter, both should work fine.